lib/jobs/generate-tag.js

```diff
@@ -41,8 +41,13 @@ function generateTagJobFactory(
     function GenerateTagJob(options, context, taskId) {
         GenerateTagJob.super_.call(this, logger, options, context, taskId);
 
-        this.nodeId = context.target || options.nodeId;
-        assert.isMongoId(this.nodeId, 'context.target || options.nodeId');
+        if (options.nodes) {
+            this.nodes = options.nodes;
+            assert.arrayOfString(this.nodes, 'options.nodes');
+        } else {
+            this.nodeId = context.target || options.nodeId;
+            assert.isMongoId(this.nodeId, 'context.target || options.nodeId');
```

Comment:
I recognize we're changing this, but should this be pinned to a mongoId or is UUID now more appropriate with the migration of node ID's to UUIDs? (or am I a clueless monkey here?)